### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by vinc3nati**
- error page
image

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
UI changes looks great
PR is good to merge

File: src/App.js [hunk 1]
```diff
@@ -24,7 +25,7 @@ function App() {
       <Routes>
         <Route path="/login" element={<Login title="login" />} />
         <Route path="/signup" element={<Signup title="register" />} />
-        <Route path="/" element={<Home />} />
+        <Route path="/" element={<Home title="home" />} />
         <Route path="/explore" element={<Explore />}>
           <Route path="" element={<VideoList title="explore" />} />
```
Comment (manish591) [inline src/App.js:30]: Here we can use `index` attribute. `""` path is not required. You can get more info about `index` from react-router docs.

**Review by manish591 — APPROVED** [review]
Code is looking good and clean.

**Review by lomesshh — APPROVED** [review]
Clean and modular code 
Looks good to merge

File: src/App.js (hunk 1 above)
Comment (vinc3nati) [reply src/App.js:30]: great, thanks

**Pull request merged**